src/unit_tests/wazuh_modules/vulnerability_detector/test_wm_vuln_detector_evr.c

```diff
@@ -128,7 +128,7 @@ void test_pkg_version_relate_epoch_VER_TYPE_unknown(void **state)
     expect_string(__wrap__mterror, tag, "wazuh-modulesd:vulnerability-detector");
     expect_string(__wrap__mterror, formatted_msg, "unknown version_type 4");
```

Comment:
This test fails. I think you have to change "unknown version_type 4" by "unknown version_type 5":
[ RUN      ] test_pkg_version_relate_epoch_VER_TYPE_unknown
[  ERROR   ] --- "unknown version_type 5" != "unknown version_type 4"

Reply:
This change was already added at https://github.com/wazuh/wazuh/pull/6709/commits/e9ff585844de4f841eecf3e3b2b3bdb8d39f1ee1#diff-f4653b27e726eb94caf5f63c77417ea7efe87052ff7a75a2b8c491c1677033d8R131.

Could you pull the branch again and verify it?